libs/service-portal/assets/src/screens/AssetsOverview/AssetsOverview.tsx

```diff
@@ -20,27 +20,6 @@ export const AssetsOverview: ServicePortalModuleComponent = () => {
         defaultMessage: `Hér eru upplýsingar um það sem kemur til með að koma inn undir
         fasteignir á næstunni.`,
       })}
-      list={{
-        title: defineMessage({
-          id: 'service.portal:incoming',
-          defaultMessage: 'Á döfinni',
-        }),
-        items: [
-          defineMessage({
-            id: 'sp.assets:inc-1',
-            defaultMessage:
-              'Yfirlit og hægt verður að greiða öll opinber gjöld',
-          }),
-          defineMessage({
-            id: 'sp.assets:inc-2',
-            defaultMessage: 'Ganga frá skattskýrsla og sjá eldi skattskýrslur',
-          }),
-          defineMessage({
-            id: 'sp.assets:inc-3',
-            defaultMessage: 'Sjá yfirlit og ráðstafa séreignarsparnaði',
-          }),
-        ],
-      }}
```

Comment:
What's happening here? How does this relate the WIP module?

Reply:
...but while we're at it: Why then is `libs/service-portal/finance/src/screens/PaymentOverview/PaymentOverview.tsx` still full of references to the `sp.assets` without ever running `useNamespaces('sp.asssets')`?

Reply:
This relates to the WIP assets module.
But you are right about the namespace. That needs to be changed.